app/_locales/en/messages.json

```
@@ -1078,7 +1078,7 @@
     "message": "Backup gas price is provided as the main gas estimation service is unavailable right now."
   },
   "eth_accounts": {
-    "message": "See address, account balance, activity and initiate transactions",
+    "message": "See address, account balance, activity and suggest transactions",
```

Another wording that seems popular in the team is to use the language of a "request"
```suggestion
    "message": "See address, account balance, activity and suggest transactions to approve",
```